Tools/TestWebKitAPI/TestWebKitAPI.xcodeproj/project.pbxproj

```diff
@@ -6567,6 +6762,22 @@
 			};
 			name = Release;
 		};
+		3A15785128D1505B00142DB1 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 3A5DDADB28D15328004DA950 /* TestWGSL.xcconfig */;
+			buildSettings = {
+				PRODUCT_NAME = "$(TARGET_NAME)";
```

Comment:
This is added by default in Xcode, but not needed, since we put everything in xcconfigs. I usually clean these up by going into the target's build settings panel, selecting "Customized", selecting all and pressing Delete.